eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/equipment/autocomplete/AutocompleteEquipmentCategory.java

```diff
@@ -25,10 +21,10 @@
 	@Path("/eqp/category/{code}")
 	@Produces("application/json")
 	@Consumes("application/json")
-	public Response complete(@PathParam("code") String code)  {
+	public Response complete(@PathParam("code") String code, @QueryParam("class") String eqpClass)  {
```

Comment:
This is subjective, consider it an optional nitpick:
@DefaultValue("") can be used here.